custom-elements/historical.html

```diff
@@ -1,35 +1,32 @@
 <!doctype html>
-<title>Custom Elements v0 features</title>
+<title>Custom Elements historical features</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
 <script>
 test(() => {
-  assert_false('registerElement' in document)
-}, 'document.registerElement should not exist')
+  assert_false('registerElement' in document);
+}, 'document.registerElement should not exist');
 
-// These tests should pass as long as v0 isn't supported:
-// v0: a 2nd string argument for createElement.
-// v1: a ElementCreationOptions (dictionary) argument.
 test(() => {
-  try {
-    const element = document.createElement('x', 'string')
-    // If neither v0/v1 are supported, then there should be no is attribute.
-    assert_false(element.hasAttribute('is'))
-  } catch (e) {
-    // If v1 is supported, then converting string to dictionary should throw.
-    assert_throws_js(TypeError, function() { throw e })
```

Comment:
This change seems like it changes the meaning of the test, which requires that `createElement('x', 'string')` throws.

Reply:
That was a bogus claim as we have ignored string arguments in the DOM standard since essentially the introduction of this feature.